packages/autotest/src/autotest/mocks/MockClassPortal.ts

```diff
@@ -116,10 +116,10 @@ export class MockClassPortal implements IClassPortal {
         return Promise.resolve({personId: null});
```

Comment:
Definitely want. Helps with mocking out AutoTest logic. ClassPortal class normally instantiated to save data to database from AutoTest run.